src/utils/track.ts

```diff
@@ -0,0 +1,130 @@
+import { Track, track as _track } from "react-tracking"
+
+// tslint:disable-next-line:no-namespace
+export namespace TrackingInfo {
+  /**
+   * The global tracking-info keys in Artsy’s schema.
+   */
+  export interface Global {
+    /**
+     * The name of an event.
+     */
+    action: string
+
+    /**
+     * The root container component should specify this as the screen context.
+     */
+    page: string
+  }
```

Comment:
Even though you would use _either_ `page` or `action` depending on if you’re decorating a class or a method, I felt like separating the two was overdoing it.

Reply:
Can you explain the `export namespace` bit?